docs/api-reference/geo-layers/tile-layer.md

```diff
@@ -268,6 +269,19 @@ Receives arguments:
 
 - `tile` (Object) - the tile that has been cleared from cache.
 
+## Tile
+
+Class to hold the reading of a single tile
+
+Properties:
```

Comment:
Missing `bbox`

Reply:
I tried to add only the most important properties, but agree this one is important